### PR TITLE
[OSDOCS#5904]: Adding links to priority ROSA troubleshooting material

### PR DESCRIPTION
**Pull request by laubai**
Adding links to priority ROSA troubleshooting material in Red Hat Knowledgebase.

Version(s):
4.12, 4.13, 4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-5904

Link to docs preview:
https://62748--docspreview.netlify.app/openshift-rosa/latest/welcome/index.html
See comments on "Files changed" tab for specifics.

QE review: N/A

Additional information: N/A

### CODE REVIEW

File: modules/life-cycle-mandatory-upgrades.adoc [hunk 1]
```diff
@@ -10,4 +10,5 @@ impacts the security or stability of the cluster, the customer must upgrade to t
...
-environment, Red Hat will notify customers that they have two link:https://access.redhat.com/articles/2623321[business days] to schedule or manually update their cluster to the latest, secure patch release. In the case that an update has not been performed, Red Hat will automatically update the cluster to the latest, secure patch release to mitigate potential security breach(es) or instability. Red Hat may, at its own discretion, temporarily delay an automated update if requested by a customer through a link:https://access.redhat.com/support[support case].
\ No newline at end of file
+environment, Red Hat will notify customers that they have two link:https://access.redhat.com/articles/2623321[business days] to schedule or manually update their cluster to the latest, secure patch release.
+In the case that an update has not been performed, Red Hat will automatically update the cluster to the latest, secure patch release to mitigate potential security breach(es) or instability. Red Hat may, at its own discretion, temporarily delay an automated update if requested by a customer through a link:https://access.redhat.com/support[support case].
```
Comment (laubai) [inline modules/life-cycle-mandatory-upgrades.adoc:14]: In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.html#rosa-mandatory-upgrades_rosa-life-cycle

File: modules/osd-applications-config-custom-domains.adoc [hunk 2]
```diff
@@ -122,3 +122,6 @@ $ oc get route -n my-project
 $ curl https://hello-openshift-tls-my-project.apps.<company_name>.io
 Hello OpenShift!
 ----
+
+.Troubleshooting
+* link:https://access.redhat.com/solutions/5419501[Error creating TLS secret]
```
Comment (laubai) [inline modules/osd-applications-config-custom-domains.adoc:127]: In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/applications/deployments/osd-config-custom-domains-applications.html

File: modules/osd-applications-renew-custom-domains.adoc [hunk 3]
```diff
@@ -32,4 +32,7 @@ $ oc patch customdomain <company_name> --type='merge' -p '{"spec":{"certificate"
 [source,terminal]
 ----
 $ oc delete secret <secret-old> -n <my_project>
-----
\ No newline at end of file
+----
+
+.Troubleshooting
+* link:https://access.redhat.com/solutions/5419501[Error creating TLS secret]
```
Comment (laubai) [inline modules/osd-applications-renew-custom-domains.adoc:38]: In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/applications/deployments/osd-config-custom-domains-applications.html#osd-applications-renew-custom-domains_osd-config-custom-domains-applications

File: modules/registry-exposing-secure-registry-manually.adoc [hunk 4]
```diff
@@ -80,3 +80,6 @@ spec:
 Only set `secretName` if you are providing a custom TLS configuration for the
 registry's route.
 ====
+
+.Troubleshooting
+* link:https://access.redhat.com/solutions/5419501[Error creating TLS secret]
```
Comment (laubai) [inline modules/registry-exposing-secure-registry-manually.adoc:85]: In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/registry/securing-exposing-registry.html#registry-exposing-secure-registry-manually_securing-exposing-registry

File: modules/rosa-deleting-cluster.adoc [hunk 5]
```diff
@@ -143,3 +143,9 @@ $ rosa delete operator-roles -c <cluster_id> --mode auto <1>
 ----
 <1> Replace `<cluster_id>` with the ID of the cluster.
 endif::sts[]
+
+.Troubleshooting
+* If the cluster cannot be deleted because of missing IAM roles, see xref:../sd_support/rosa-troubleshooting-deployments.adoc#rosa-troubleshooting-cluster-deletion_rosa-troubleshooting-cluster-deployments[Repairing a cluster that cannot be deleted].
+* If the cluster cannot be deleted for other reasons:
+** Check that there are no Add-ons for your cluster pending in the link:https://console.redhat.com/openshift[Hybrid Cloud Console].
+** Check that all AWS resources and dependencies have been deleted in the Amazon Web Console.
```
Comment (laubai) [inline modules/rosa-deleting-cluster.adoc:151]: In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.html#rosa-deleting-cluster_rosa-sts-deleting-cluster

File: rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc [hunk 6]
```diff
@@ -72,6 +72,7 @@ endif::[]
 
 [id="nodes-about-autoscaling-nodes-additional-resources"]
 == Additional resources
+* link:https://access.redhat.com/solutions/6821651[Troubleshooting: Autoscaling is not scaling down nodes]
```
Comment (laubai) [inline rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc:75]: In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html#nodes-about-autoscaling-nodes-additional-resources

File: modules/life-cycle-mandatory-upgrades.adoc (hunk 1 above)
Comment (nalhadef) [inline modules/life-cycle-mandatory-upgrades.adoc:14]: I may be misunderstanding these two paragraphs. In the first, the user is notified that they have two business days to update. In the second paragraph if an update has not been done. it happens automatically. Can there be more of a differentiation between these two situations?

**Review by nalhadef — COMMENTED** [review]
Hi, @laubai. Looks good. I had just one question.

File: modules/life-cycle-mandatory-upgrades.adoc [hunk 7]
```diff
@@ -10,4 +10,5 @@ impacts the security or stability of the cluster, the customer must upgrade to t
 patch release within two link:https://access.redhat.com/articles/2623321[business days].
```
[the remark this answers is not in the document]
Comment (laubai) [reply modules/life-cycle-mandatory-upgrades.adoc:10]: Ah, I wonder how this has hidden for so long? Thanks!

File: modules/life-cycle-mandatory-upgrades.adoc (hunk 1 above)
Comment (laubai) [reply modules/life-cycle-mandatory-upgrades.adoc:14]: I believe they receive two days' warning to do the upgrade themselves, after which Red Hat updates the cluster, but I'll confirm with SRE.

Comment (laubai) [reply modules/life-cycle-mandatory-upgrades.adoc]: Verified with SRE and updated to clarify. Thanks!

**Comment by stevsmit** [comment]
This content looks good to me.

**Pull request merged**